com.unity.render-pipelines.core/Runtime/Debugging/DebugUpdater.cs

```diff
@@ -4,22 +4,27 @@
 using UnityEngine.InputSystem.UI;
 using UnityEngine.InputSystem.EnhancedTouch;
 #endif
+using System;
 using System.Collections;
 using UnityEngine.EventSystems;
 
 namespace UnityEngine.Rendering
 {
     class DebugUpdater : MonoBehaviour
     {
+        static DebugUpdater s_Instance = null;
+
         ScreenOrientation m_Orientation;
+        bool m_RuntimeUiWasVisibleLastFrame = false;
 
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.AfterSceneLoad)]
         static void RuntimeInit()
         {
-            if (!Debug.isDebugBuild || !DebugManager.instance.enableRuntimeUI || FindObjectOfType<DebugUpdater>() != null)
+            if (!Debug.isDebugBuild)
```

Comment:
In another PR you have another system with some defines to check if it is a debug build, do we want to have this here like this?

Reply:
I think I'll just reconcile these changes into the other PR, I'll need to do a merge there anyway.